src/main/kotlin/org/elm/lang/core/resolve/scope/GlobalScope.kt

```diff
@@ -88,7 +88,8 @@ class GlobalScope private constructor(val clientFile: ElmFile) {
         return rest
     }
 
-
+    // this function should also return List, but, unlike all the other global types, Elm doesn't
```

Comment:
Not having `type List = List` anywhere causes so many problems.

Reply:
I know. I asked Evan if he could fix it but got no response. I'll bring it up again.

Reply:
I created an issue for it https://github.com/elm/core/issues/1037